### PR TITLE
Fix link for switching between overview thresholds

### LINKED ISSUE #356: Changed coverage links on overview page broken
**Describe the bug**

On the overview report page, the "Completeness" links are broken (i.e. "Completeness 30x" in the screenshot below):

incomplete_genes_overview

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added/Changed/Fixed
- Tentative fix #356

### How to test
- [ ] Deploy on stage
- [ ] Switch between thresholds on overview report

### Expected outcome
- The links should work

## Review
- [ ] Tests executed by 
- [ ] "Merge and deploy" approved by

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: src/chanjo2/templates/overview.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 
 {% macro incompletely_covered_intervals(active_level) %}
- <form id="geneStatsForm" action="{{url_for('gene_overview')}}" method="post">
+ <form id="geneStatsForm" action="{{url_for('overview')}}" method="post">
```
Comment (northwestwitch) [inline src/chanjo2/templates/overview.html:24]: This link was wrong and not used

File: src/chanjo2/templates/overview.html [hunk 2]
```diff
@@ -108,7 +108,6 @@ <h4>Incomplete {{ extras.interval_type }}</h4>
 		function updateOverview(customLevel)
 		{
 			document.getElementById("defaultLevel").value = customLevel;
-			theForm.action = '/overview';
```
Comment (northwestwitch) [inline src/chanjo2/templates/overview.html:111]: This shouldn't be necessary if the form action link is correct

File: src/chanjo2/templates/overview.html [hunk 3]
```diff
@@ -47,7 +47,7 @@
 					{% for row in incomplete_coverage_rows %}
 						<tr>
 							<td>
-								<a href="#" onclick="getGeneStatsPage({{row[1]}});" target="_blank" rel="noopener">
```
Comment (northwestwitch) [inline src/chanjo2/templates/overview.html:50]: This didn't work for opening the gene report on a new tab

File: src/chanjo2/templates/overview.html [hunk 4]
```diff
@@ -120,6 +120,8 @@ <h4>Incomplete {{ extras.interval_type }}</h4>
 			geneInput.setAttribute("name", "hgnc_gene_id");
 			geneInput.setAttribute("value", hgncId);
 			theForm.appendChild(geneInput);
+			theForm.setAttribute("target", "_blank");
```
Comment (northwestwitch) [inline src/chanjo2/templates/overview.html:123]: This way it works

File: src/chanjo2/templates/overview.html [hunk 5]
```diff
@@ -108,7 +108,7 @@ <h4>Incomplete {{ extras.interval_type }}</h4>
 		function updateOverview(customLevel)
 		{
 			document.getElementById("defaultLevel").value = customLevel;
-			theForm.action = '/overview';
+			theForm.action = '{{url_for("overview")}}';
```
Comment (northwestwitch) [inline src/chanjo2/templates/overview.html:111]: This is the fix

**Comment by Jakob37** [comment]
I'll test!

**Review by Jakob37 — APPROVED** [review]
Works nicely! The coverage buttons works and the gene opens in a new page.

30x

The coverage depth buttons also opens a new tab. Which might be unexpected? (Fine either way with me.)

Nice 👌

**Comment by northwestwitch** [comment]
> The coverage depth buttons also opens a new tab. Which might be unexpected? (Fine either way with me.)

AH right, it shouldn't. I'll fix so only genes open. in a new tab

**Comment by northwestwitch** [comment]
Tested locally and now only genes open to new tabs. Thanks for testing @Jakob37 - merging!

**Pull request merged**